sbncode/EventGenerator/MeVPrtl/config/numi_kaon_common.fcl

```diff
@@ -5,15 +5,14 @@ empty_kaon: {
 }
 
 # Taken from: https://github.com/SBNSoftware/icaruscode/blob/develop/fcl/gen/numi/GNuMIFlux.xml#L765
-icarus_numi_beam2detector_rotation: [   0.921228671,    0.0226872648,   0.388359401,
-				        0.00136256111,  0.998103714,   -0.061539578,
-				       -0.389019125,    0.0572211871,   0.919450845  ]
+icarus_numi_beam2detector_rotation: [ 0.921035925, 0.022715103,  0.388814672,
+                                                0, 0.998297825, -0.058321970
```

Comment:
@gputnam @miquelnebot this is where the error crept in, no comma at end of this line